dev-docs/bidders/appush.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: bidder
+title: Appush
+description: Prebid Appush Bidder Adapter
+biddercode: appush
+media_types: banner, video, native
+pbjs: false
+pbs: true
+gvl_id: 879
```

Comment:
If possible, could you add the additional fields. It helps publisher and sales houses to evaluate your adapter.

* `safeframes_ok`: `true` or `false`
* `floors_supported`: `true` or `false`
* `fpd_supported`: `true` or `false`
* `multiformat_supported`: will-bid-on-any, will-bid-on-one, will-not-bid
* `ortb_blocking_supported`: true/partial/false

Reply:
Thanks a lot 🤗